### DISCUSSION TITLE
Looking to emulate IBM PC 5150 (DOS 2.10+) and connect to real IBM 5153 CGA monitor

### DISCUSSION THREAD

**Discussion by markostamcar** [fdivitto/FabGL#264] [Q&A]
Hello, thanks for a great project! Here at the Slovenian Computer Museum we have a demonstration of the original IBM PC model 5150 connected to a CGA monitor IBM model 5153. The computer is running DOS 2.10 and a specific software demonstration running in an infinite loop without any need for keyboard/mouse.

Since this computer is a very old machine, I would like to retire it from being powered on as to not stress it and perhaps replace the "computer" part with ESP32 running a PC emulator. What would be great is to use the minimal hardware possible, perhaps a NodeMCU dev board and use the GPIO to generate CGA TTL image. Is this something possible with your project?

Thank you so much for your work and support!

![ibm5150](https://user-images.githubusercontent.com/1268630/176993653-116bdc29-66c1-47a2-9a71-7dde0e40751c.jpg)

**Comment by fdivitto** [comment]
Hello,
at the moment TTL CGA is not supported. It could be possible, but I cannot support or develop it because unfortunately I haven't a CGA monitor for tests.

**Reply by markostamcar** [reply]
Thanks for your reply! I can gladly donate towards your project and supply you with a adapter such as this one:
https://www.tindie.com/products/gglabs/cga2rgbv2-digital-to-analog-rgb-for-mdacgaega/

**Reply by lgpf** [reply]
But that device would require a monitor that supports 15khz horizontal sync right?

**Reply by markostamcar** [reply]
The main idea is to get an original IBM 5153 monitor running. We have since found another way to do it using a RPi:
https://github.com/trylle/ibm515x/issues/3
